test/unit/org/apache/cassandra/db/streaming/CassandraEntireSSTableStreamWriterTest.java

```diff
@@ -112,7 +111,7 @@ public void testBlockWriterOverWire() throws IOException
     {
         StreamSession session = setupStreamingSessionForTest();
 
-        CassandraEntireSSTableStreamWriter writer = new CassandraEntireSSTableStreamWriter(sstable, session, CassandraOutgoingFile.getComponentManifest(sstable));
+        CassandraEntireSSTableStreamWriter writer = new CassandraEntireSSTableStreamWriter(sstable, session, ComponentManifest.create(sstable, false));
 
         EmbeddedChannel channel = new EmbeddedChannel();
```

Comment:
@jasonstack Should we still close this channel? (It looks like we more or less reverted this back to what's in trunk, which is fine, but wanted to make sure we didn't throw the baby out with the bath water...)

Reply:
+1